src/index.hbs

```diff
@@ -7,8 +7,107 @@
       content='{{htmlWebpackPlugin.options.description}}'
     />
     <meta name='viewport' content='width=device-width, initial-scale=1' />
```

Comment:
- [ ] for accessibility purposes kindly update your website title with a descriptive title , currently its displaying 'Webpack Template '